winter_ddd/domain_event_dispatcher.py

```diff
@@ -9,11 +12,13 @@
 T = TypeVar('T')
 HandlerFactory = Callable[[Type[T]], T]
 
+EventFilter = Tuple[Type[DomainEvent]]
+
 
 class DomainEventDispatcher:
     def __init__(self):
-        self._subscriptions = {}
-        self._event_type_to_event_filter_map = {}
+        self._subscriptions: Dict[EventFilter, List[DomainEventSubscription]] = {}
+        self._event_type_to_event_filter_map: Dict[Type[DomainEvent], List[EventFilter]] = {}
```

Comment:
```suggestion
        self._event_type_to_event_filters_map: Dict[Type[DomainEvent], List[EventFilter]] = {}
```
Там список, поэтому кажется надо добавить s